cigaradvisor/blocks/article-teaser/article-teaser.js

```diff
@@ -30,6 +30,7 @@ export function buildArticleTeaser(parentElement, article) {
   };
 
   const picture = createOptimizedPicture(article.image);
+  picture.querySelector('img').setAttribute('alt', article.path.substring(article.path.lastIndexOf('/') + 1).replace(/-/g, ' '));
```

Comment:
This should actually be the article's title, not the URL value.

Reply:
changed to heading.

Reply:
felt heading is more readable than title